src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResultSetMetaData.java

```diff
@@ -125,6 +125,14 @@ public int getColumnType(int column) throws SQLServerException {
         if ( SSType.SQL_VARIANT == typeInfo.getSSType()){
             jdbcType = JDBCType.SQL_VARIANT;
         }
+        if (SSType.UDT == typeInfo.getSSType()) {
+            if (typeInfo.getSSTypeName().equalsIgnoreCase("geometry")) {
```

Comment:
I don't really have a problem with this, but why are we doing type check in 3 different ways. VARIANT is being checked separately outside the switch, and now GEOMETRY/GEOGRAPHY will be checked by name. Can we maybe stick to one form? This Microsoft [overview](https://docs.microsoft.com/en-us/sql/relational-databases/spatial/spatial-data-types-overview) of spatial datatypes seems to imply that it's only available in SQL Server 2012 and later, so shouldn't this be in the isKatamaiOrLater() block?

Reply:
We can't stick to one form. If you debug this part of the code, you'll see that the getSSType will give SQL_VARIANT or UDT. If it's UDT, we need to look at the SSTypeName to find out if it's Geometry or Geography, so this is the only way to check for these datatypes.

The overview page mentions that new spatial features have been introduced in SQL Server 2012, but this does not mean that spatial datatypes are supported starting from 2012 - it's supported starting from 2008. so we need this check for 2008 and up.

Reply:
Please create Constants for these strings and reference them here.